### PR TITLE
ES6ify the code

### LINKED ISSUE #18: ES6ify the code.
The jscodeshift CLI automatically loads `babel/register` (it's still on `babel@5`).

No reason we shouldn't be able to do the same for our tests, etc.

### PR DESCRIPTION
**Pull request by sotojuan**
Fixes #18.

### CODE REVIEW

**Comment by sotojuan** [comment]
Fixing the cause of the failure (forgot about it!) 😁

**Comment by jamestalmage** [comment]
Hmm.

I wasn't doing `use-strict`. I wonder if that is your problem somehow.

Use strict is implied for ES2015 modules. Not sure why Babel doesn't just handle that.

File: cli-utils.js [hunk 1]
```diff
@@ -30,21 +28,17 @@ function getVersions(codemods) {
...
+	return flatten(scripts).map(script => path.join(__dirname, script));
 }
 
 module.exports = {
-	sortByVersion: sortByVersion,
-	getVersions: getVersions,
-	selectScripts: selectScripts
+	sortByVersion,
+	getVersions,
+	selectScripts
```
Comment (sindresorhus) [inline cli-utils.js]: Use ES2015 `export`.

File: cli-utils.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 'use strict';
```
Comment (sindresorhus) [inline cli-utils.js]: Remove all the strict directives. Not needed.

File: cli.js [hunk 3]
```diff
@@ -96,21 +96,19 @@ var questions = [{
 	message: 'On which files should the codemods be applied?',
 	default: (avaConf.files && arrify(avaConf.files).join(' ')) || defaultFiles,
 	when: !cli.input.length,
-	filter: function (files) {
-		return files.trim().split(/\s+/).filter(function (v) {
-			return v;
-		});
+	filter: files => {
+		return files.trim().split(/\s+/).filter(v => v);
```
Comment (sindresorhus) [inline cli.js]: Can be an inline arrow expression.

File: lib/utils.js [hunk 4]
```diff
@@ -4,10 +4,7 @@ exports.renameAssertion = function (name, newName, j, ast) {
 	ast.find(j.CallExpression, {
```
Comment (sindresorhus) [inline lib/utils.js:2]: ES2015 `export` above here.

File: lib/same-to-deep-equal.js [hunk 5]
```diff
@@ -1,9 +1,9 @@
 'use strict';
-var utils = require('./utils');
+const utils = require('./utils');
 
 module.exports = function transformer(file, api) {
```
Comment (sindresorhus) [inline lib/same-to-deep-equal.js]: ES2015 `export` here.

File: lib/same-to-deep-equal.js [hunk 6]
```diff
@@ -1,9 +1,9 @@
 'use strict';
-var utils = require('./utils');
+const utils = require('./utils');
```
Comment (sindresorhus) [inline lib/same-to-deep-equal.js]: `import`

File: lib/ok-to-truthy.js [hunk 7]
```diff
@@ -1,9 +1,9 @@
 'use strict';
-var utils = require('./utils');
+const utils = require('./utils');
```
Comment (sindresorhus) [inline lib/ok-to-truthy.js]: `import`

File: lib/ok-to-truthy.js [hunk 8]
```diff
@@ -1,9 +1,9 @@
...
 
 module.exports = function transformer(file, api) {
```
Comment (sindresorhus) [inline lib/ok-to-truthy.js]: ES2015 `export` here.

File: cli.js [hunk 9]
```diff
@@ -51,22 +51,22 @@ var cli = meow([
...
 if (!clean) {
 	if (cli.flags.force) {
-		console.log('WARNING: ' + errorMessage + '. Forcibly continuing.');
+		console.log(`WARNING: ${errorMessage}. Forcibly continuing.`);
 		console.log(ENSURE_BACKUP_MESSAGE);
 	} else {
-		console.log('ERROR: ' + errorMessage + '. Refusing to continue.');
+		console.log(`ERROR: ${errorMessage}. Refusing to continue.`);
 		console.log(ENSURE_BACKUP_MESSAGE);
 		console.log('You may use the --force flag to override this safety check.');
```
Comment (sindresorhus) [inline cli.js]: Instead of many console.log calls here, just do one with everything.

File: cli.js [hunk 10]
```diff
@@ -51,22 +51,22 @@ var cli = meow([
...
 
-var clean = false;
-var errorMessage = 'Unable to determine if git directory is clean';
+let clean = false;
+let errorMessage = 'Unable to determine if git directory is clean';
 try {
 	clean = isGitClean.sync();
 	errorMessage = 'Git directory is not clean';
 } catch (e) {
 }
```
Comment (sindresorhus) [inline cli.js:59]:
``` js
} catch (err) {}
```

While you're at it.

File: cli.js [hunk 11]
```diff
@@ -31,7 +31,7 @@ function runScripts(scripts, files) {
 	});
 }
 
-var cli = meow([
+const cli = meow([
 	'Usage',
```
Comment (sindresorhus) [inline cli.js]: The `meow` help should use a template literal. See: https://github.com/sindresorhus/trash-cli/blob/e6dee38aa399758f6d2a66fa0cb56b052f327cfa/cli.js#L7

File: cli.js [hunk 12]
```diff
@@ -1,28 +1,28 @@
...
+const assign = require('lodash.assign');
+const npmRunPath = require('npm-run-path');
+const isGitClean = require('is-git-clean');
+const utils = require('./cli-utils');
+const codemods = require('./codemods.json');
 
 function runScripts(scripts, files) {
-	var spawnOptions = {
+	const spawnOptions = {
 		env: assign({}, process.env, {PATH: npmRunPath({cwd: __dirname})}),
```
Comment (sindresorhus) [inline cli.js:18]: Use `Object.assign`.

File: cli.js [hunk 13]
```diff
@@ -1,28 +1,28 @@
...
-var arrify = require('arrify');
-var globby = require('globby');
-var pkgConf = require('pkg-conf');
-var inquirer = require('inquirer');
-var assign = require('lodash.assign');
-var npmRunPath = require('npm-run-path');
-var isGitClean = require('is-git-clean');
-var utils = require('./cli-utils');
-var codemods = require('./codemods.json');
+const childProcess = require('child_process');
```
Comment (sindresorhus) [inline cli.js]: Change out this with [`execa`](https://github.com/sindresorhus/execa) while you're at it. You can then remove the `npmRunPath` stuff as it's built into `execa`.

File: cli.js [hunk 14]
```diff
@@ -1,28 +1,28 @@
 #!/usr/bin/env node
 'use strict';
 global.Promise = require('pinkie-promise');
```
Comment (sindresorhus) [inline cli.js]: This can just be `global.Promise = Promise;`.

File: cli.js [hunk 15]
```diff
@@ -1,28 +1,28 @@
...
+const updateNotifier = require('update-notifier');
+const arrify = require('arrify');
+const globby = require('globby');
+const pkgConf = require('pkg-conf');
+const inquirer = require('inquirer');
+const assign = require('lodash.assign');
+const npmRunPath = require('npm-run-path');
+const isGitClean = require('is-git-clean');
+const utils = require('./cli-utils');
+const codemods = require('./codemods.json');
```
Comment (sindresorhus) [inline cli.js]: ES2015 `import`

**Comment by sotojuan** [comment]
All right, phew, fixed a lot of stuff. The rest should just be style fixes from now on.

**Comment by jamestalmage** [comment]
What was the issue? The `use strict` line?

**Comment by sotojuan** [comment]
That and I forgot to add more files to the `only` array in the Babel config 😅

**Comment by sotojuan** [comment]
@sindresorhus What do you think?

**Comment by sindresorhus** [comment]
Looks good. Thanks @sotojuan :)

**Pull request merged**